crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -28,1847 +28,1793 @@ use std::{env, str::FromStr};
 
 const MAX_GAS: u64 = 10000;
 
-fn run_tokio_test_with_big_stack<F>(fut: F)
-where
-    F: std::future::Future<Output = ()> + Send + 'static,
-{
-    let runtime = tokio::runtime::Builder::new_multi_thread()
-        .enable_time()
-        .thread_stack_size(128 * 1024 * 1024)
-        .build()
-        .unwrap();
-    runtime.block_on(async move {
-        // spawn future on a worker thread with a big stack
-        let handle = tokio::task::spawn(fut);
-        handle.await.unwrap()
-    })
+#[tokio::test]
+#[cfg_attr(msim, ignore)]
+async fn test_object_wrapping_unwrapping() {
```

Comment:
The main change here is no longer wrapping these tests with `run_tokio_test_with_big_stack`, but it shows up as a big diff because of whitespace changes.

Reply:
You're probably aware but there's always the `Hide whitespace` checkbox in GH that makes this kind of thing more digestible 😄 I often forget to toggle when reviewing PRs

<img width="362" alt="Screen Shot 2022-12-13 at 6 24 34 PM" src="https://user-images.githubusercontent.com/888624/207490059-8575280d-cd1c-49e3-af12-63da9d0bcc9d.png">